spring-boot-project/spring-boot-starters/README.adoc

```diff
@@ -161,7 +161,7 @@ do as they were designed before this was clarified.
 | https://github.com/liquigraph/liquigraph
 
 | https://logback.qos.ch/access.html[Logback-access]
-| https://github.com/akihyro/logback-access-spring-boot-starter
+| https://github.com/akkinoc/logback-access-spring-boot-starter
```

Comment:
Is there any relationship between akihyro's starter and akkinoc's starter?

Reply:
I do not really know, but older commits are from Akihiro Kondo (https://github.com/akkinoc/logback-access-spring-boot-starter/commits/main?after=570258387eb0f5dec348f1b0df505a67a890b5cc+314&branch=main&qualified_name=refs%2Fheads%2Fmain), it is active (https://github.com/akkinoc/logback-access-spring-boot-starter/pulse), up-to-date and popular.

Maybe you should check with @akkinoc ?

Reply:
Thank you for fixing the link.
I changed my username from akihyro to akkinoc a few years ago.
akihyro and akkinoc are the same person.

Reply:
Thank you, @akkinoc.